src/foam/nanos/controller/ApplicationController.js

```diff
@@ -512,6 +513,14 @@ foam.CLASS({
       if ( ! lastTheme || lastTheme.id != this.theme.id ) this.useCustomElements();
     },
 
+    //TODO add the option to the user to select a language
```

Comment:
default considerations - we can provide a default for a domain Theme, then Group, and finally the User.